File: sample-infrastructure-services-domain/npm/src/dependency-injection/options.ts
export interface IOptions { 
    enableQueries: boolean;
    enableCommands: boolean;
}

export class Options implements IOptions { 
    public enableQueries: boolean;
    public enableCommands: boolean;
    constructor() {
        this.enableQueries = true;
        this.enableCommands = true;
    }
}

File: sample-infrastructure-services-domain/dotnet/src/dependency-injection/options.cs
namespace Sample.Infrastructure.Services.Domain.DependencyInjection
{
    public interface IOptions 
    { 
        bool EnableQueries { get; set; }
        bool EnableCommands { get; set; }
    }
    
    public class Options : IOptions 
    { 
        public bool EnableQueries { get; set; } = true;
        public bool EnableCommands { get; set; } = true;
    }
}